asserts/device_asserts.go

```diff
@@ -194,3 +194,61 @@ func assembleSerial(assert assertionBase) (Assertion, error) {
 		pubKey:        pubKey,
 	}, nil
 }
+
+// SerialRequest holds a serial-request assertion, which is a self-signed request to obtain a full device identity bound to the device public key.
+type SerialRequest struct {
+	assertionBase
+	pubKey PublicKey
+}
+
+// BrandID returns the brand identifier of the device making the request.
+func (sreq *SerialRequest) BrandID() string {
+	return sreq.HeaderString("brand-id")
+}
+
+// Model returns the model name identifier of the device making the request.
+func (sreq *SerialRequest) Model() string {
+	return sreq.HeaderString("model")
+}
+
+// NonceTicket returns the nonce/ticket for the request.
+func (sreq *SerialRequest) NonceTicket() string {
+	return sreq.HeaderString("nonce-ticket")
```

Comment:
Do we really need "nonce" and "ticket" in the name of this header? tbh, tbh, they're both fairly meaningless terms anyway, so I wonder if something more like "authentication-nonce" might be better anyway?